src/app/global/components/sidepanel/sidepanel.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { simpleFadeIn, inOutAnimation } from '../../animations/animations';
+import { parentFadeIn, slideInOutAnimation } from '../../animations/animations';
+
+interface NamedItem {
+  name: string;
+}
+
+interface Option {
+  label: string;
+  icon: string;
+  url: string;
+}
+
+@Component({
+  selector: 'unfetter-side-panel',
+  templateUrl: './sidepanel.component.html',
+  styleUrls: ['./sidepanel.component.scss'],
+  animations: [simpleFadeIn, inOutAnimation, parentFadeIn, slideInOutAnimation]
+})
+export class SidepanelComponent implements OnInit {
+
+    @Input() private width = 320;
```

Comment:
material suggested max width at 400px, should we change from 320?

Reply:
I based that width on the ADG wireframe. I can change it to 400 (is there a constant somewhere I can use rather than a hard-coded number?), and alter the threat-dashboard html to specify 320.